### PR TITLE
Adding Eclipse Support or .Jar file for SDK.

### PR DESCRIPTION
**Pull request by aamir**
- Added additional file for jar with dependencies, just incase. 
- The final version in the repo should be sdk-1.1.4-jar-without-dependencies.jar.

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -27,12 +27,11 @@ android {
 
 
 dependencies {
-    compile 'com.android.support:appcompat-v7:21.0.3'
```
Comment (jerluc) [inline android/build.gradle]: Was this just unused?

Comment (aamir) [inline android/build.gradle]: Yeah, well we did not need it in the first place. It was some goof up earlier. Became more evident when we needed to create Jar's.

File: README.md [hunk 2]
```diff
@@ -21,6 +26,24 @@ compile 'com.urx.android:sdk:1.1.+'
 </dependency>
 ```
 
+#### Eclipse Integration.
+We strongly advise to try to move to Android Studio. However if Eclipse is what you need to get started. We have a created a jar as well to use with Eclipse. Please follow 
+the these steps.
```
Comment (andrewlook) [inline README.md]: I would word this as: "We strongly advise developers to develop with Android Studio. However, we have packaged our SDK as a jar as well to use with Eclipse. Please follow the steps below to get started."

File: README.md [hunk 3]
```diff
@@ -21,6 +26,24 @@ compile 'com.urx.android:sdk:1.1.+'
...
 ```
 
+#### Eclipse Integration.
+We strongly advise to try to move to Android Studio. However if Eclipse is what you need to get started. We have a created a jar as well to use with Eclipse. Please follow 
+the these steps. 
+
+##### Step 1 : Download the jar from the [Maven repository](http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22com.urx.android%22%20AND%20a%3A%22sdk%22). Make sure to download
+the .jar and not the .aar file. 
+
+##### Step 2 : Use the above jar and add it The above jar does not have all of its dependencies. You would have to manually add all the dependencies required by the URX SDK. Attached
```
Comment (andrewlook) [inline README.md]: I would make sure the header lines don't get too long here

**Comment by andrewlook** [comment]
LGTM!

**Pull request merged**